packages/grafana-data/src/types/datasource.ts

```diff
@@ -552,6 +556,9 @@ export interface DataQueryRequest<TQuery extends DataQuery = DataQuery> {
   panelId?: number;
   dashboardUID?: string;
 
+  /** Filters to apply to all queries */
+  adhocFilters?: AdHocVariableFilter[];
```

Comment:
Just a thought- maybe we could just name it `filters` to decouple from the feature name (ad hoc filters)?

Reply:
@dprokop don't we want them to be coupled? so it's clear that AdHocFiltersSet results in this array? makes the transition a bit more intuitive (templateSrv.getAdhocFilters => request.adhocFilters). 

the code comments for getTagKeys and getTagValues mention adhoc filters, so it's the term we use everywhere. but I like filters as well (shorter and easier to type) so no strong feelings either way.

Reply:
@dprokop 
after thinking more I renamed property (and function args) to just filters